### PR TITLE
Ht1 data changed

### PR DESCRIPTION
**Pull request by Lyubov-smile**

### CODE REVIEW

File: Data/task05.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# 5. Write a Python program to check if a string starts with "if".
+
+s = str(input('Input a string: '))
+print(s[0:2] == 'if')
```
Comment (SofiaNabyvantes) [inline Data/task05.py]: you can use` result = s.startswith('if') `too

File: Data/task08.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# 8. Write a Python program to check three numbers and return true if one or more of them are small.
+# A number is called "small" if it is in the range 1..10 inclusive. 
+
+a = int(input())
+b = int(input())
+c = int(input())
+if 1 <= a <= 10:
```
Comment (SofiaNabyvantes) [inline Data/task08.py]: one more of the variant:  2 in range(1,11)

File: Data/task24.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+a2 = 2.3
+a3 = dt1
+print(a1, a2, a3)
+ar = [a1, a2, a3]
+print(ar)
+
+#ar = ar + [dt]
+#a = ["Ruby", 2.3, % now.hour, % now.minute"]

+#print(a[0],a[1],a[2])
```
Comment (SofiaNabyvantes) [inline Data/task24.py]: please, print elements of array using loop(e.g for)

File: Data/task17.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# 17. Write a Python program to check if a number is within 10 of 100 or 200. (E.g. 90, 110, 190, 210) 
+
+a = int(input('Input number: '))
+
+if abs(100 - a) <= 10 or abs(200 - a) <= 10:
```
Comment (SofiaNabyvantes) [inline Data/task17.py]: So in this task, you need to check if *a* in range from 10 to 100 or in range from 10 to 200

File: Data/task21.py [hunk 5]
```diff
@@ -0,0 +1,6 @@
+# 21. Write a Python program to print 34 upto 41.
+
+a = int(34)
```
Comment (SofiaNabyvantes) [inline Data/task21.py]: 3 and 4 line don't needed

File: Data/task21.py [hunk 6]
```diff
@@ -0,0 +1,6 @@
...
+b = int(41)
+for i in range(a, b+1, 1):
```
Comment (SofiaNabyvantes) [inline Data/task21.py]: then `for i n range(34, 42)`:

File: Data/task26.py [hunk 7]
```diff
@@ -0,0 +1,19 @@
+# 26. Write a Python program to retrieve the total marks where subject name and marks of a student stored in a hash.
+
+subject_marks = {"Literature": 74, "Science": 89, "Math": 91}
+
+def Sum(myDict):
+    s = 0
+    for i in myDict:
```
Comment (SofiaNabyvantes) [inline Data/task26.py]: one more way to do it:
```
for name, mark in subject_marks.items():
  s += mark
```

File: Data/task17.py (hunk 4 above)
Comment (Lyubov-smile) [reply Data/task17.py]: I understood it after I saw an example in internet.

File: Data/task21.py (hunk 5 above)
Comment (Lyubov-smile) [reply Data/task21.py]: Should I delete them here or on laptop?

File: Data/task21.py (hunk 6 above)
Comment (Lyubov-smile) [reply Data/task21.py]: I wrote 3 and 4 line if we decide to change a and b.

File: Data/task26.py (hunk 7 above)
Comment (Lyubov-smile) [reply Data/task26.py]: Before it I need to write s = 0. Now I understood how items() method works.

File: Data/task24.py (hunk 3 above)
Comment (Lyubov-smile) [reply Data/task24.py]: I don't understand: should I do this program for my version of Python (in this case I don't now how to do it) or should I do it for given data ["Ruby", 2.3, Time.now] 
I made task24_2.py

import datetime

arr = []
arr.append("Ruby")
arr.append(2.3)
arr.append(float(datetime.datetime.now().strftime("%H.%M")))
print (arr)
for i in range(len(arr)):
    print(arr[i])

File: Data/task08.py (hunk 2 above)
Comment (Lyubov-smile) [reply Data/task08.py]: How to add this variant into the file here on github.com?

File: Data/task05.py (hunk 1 above)
Comment (Lyubov-smile) [reply Data/task05.py]: Can you change the program or only leave a comment? It's interesting how it looks like if you add these two lines:
result = s.startswith('if')
print(result)

**Pull request merged**